patterns/1-initial/get-contributor-despite-silo.md

```diff
@@ -0,0 +1,105 @@
+## Title
+
+Incentive mechanisms to foster voluntary contribution
```

Comment:
The pattern title does not match the file name. This is probably a good time to clean that up, as well.

Reply:
Fixed in 60e1f82